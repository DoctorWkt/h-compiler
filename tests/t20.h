void main()
{
  int x=1;
  int y;

  y = x << 1; printn(y);
  y = x << 2; printn(y);
  y = x << 3; printn(y);
  y = x << 4; printn(y);
  y = x << 5; printn(y);
  y = x << 6; printn(y);
  y = x << 7; printn(y);
}
