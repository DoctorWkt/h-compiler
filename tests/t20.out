000002
000004
000010
000020
000040
000100
000200
